guides/release/in-depth-topics/autotracking-in-depth.md

```diff
@@ -180,16 +202,26 @@ app.
 So far we've only shown tracked properties working through getters, but tracking
 works through _methods_ or _functions_ as well:
 
-```js
+``` gjs { data-diff="+17,+18,+19,+20,+21,+24,+25,+26,+27" }
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import { action } from '@ember/object';
+import { on } from '@ember/modifier';
 
 export default class HelloComponent extends Component {
+  constructor() {
```

Comment:
we should probably state somewhere that folks shouldn't use constructors in real code, except in very special circumstances

Reply:
and also that if you do async behavior like this, you have to choose your own adventure for loading/error/etc state handling

Reply:
I disagree on the constructor use. 

I do agree the behavior of this async isn't a great example, but it is building on stuff introduced before and would require a larger rewrite.